### PR TITLE
Deploy etcd with tls and StorageOS with tls

### PR DESCRIPTION
**Pull request by avestuk**

### CODE REVIEW

File: k8s/deploy-storageos/etcd-tls/storageos-deployment/manifests/040_daemonset.yaml_template [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            mountPath: /sys
+            mountPropagation: Bidirectional
+        securityContext:
+          privileged: true
+          capabilities:
+            add:
+            - SYS_ADMIN
+      containers:
+      - name: storageos
+        image: "soegarots/node:abce45e-ff9f7c2c"
```
Comment (Arau) [inline k8s/deploy-storageos/etcd-tls/storageos-deployment/manifests/040_daemonset.yaml_template]: image tag not approriate.

**Review by Arau — CHANGES_REQUESTED** [review]
Image tag pointing to wrong repo.

I haven't reviewed the whole PR yet.

File: k8s/deploy-storageos/etcd-tls/deploy-etcd-tls.sh [hunk 2]
```diff
@@ -0,0 +1,66 @@
+ETCD_NS=etcd
```
Comment (darkowlzz) [inline k8s/deploy-storageos/etcd-tls/deploy-etcd-tls.sh:1]: Add shebang and `set -e`?

File: k8s/deploy-storageos/etcd-tls/storageos-deployment/manifests/040_daemonset.yaml_template (hunk 1 above)
Comment (avestuk) [reply k8s/deploy-storageos/etcd-tls/storageos-deployment/manifests/040_daemonset.yaml_template]: I'll leave this outstanding until we have a release container ready

**Comment by Arau** [comment]
Closing PR and repushing the changes to a new PR

**Pull request closed without merging**